include/range/v3/utility/semiregular.hpp

```diff
@@ -195,6 +195,13 @@ namespace ranges
         using semiregular_t =
             meta::if_<SemiRegular<T>, T, semiregular<T>>;
 
+        template<typename T>
+        using movesemiregular_t =
+            meta::if_c<
+                Movable<T>() && DefaultConstructible<T>(),
+                T,
+                semiregular<T>>;
```

Comment:
Same. Hate it utterly, have no better idea, was sure you would know what this name means.